components/modal/EditRequest.tsx

```diff
@@ -0,0 +1,128 @@
+                      <p className="font-bcsans text-sm text-gray-900">
+                        Please provide your final comments to be shared.
+                      </p>
+                    </div>
+                  </div>
+                </div>
+                <p className="mt-12 font-bcsans text-sm text-gray-900 font-bold">Comments</p>
+                <textarea
+                  onChange={handleCommentChange}
+                  className="w-full border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-slate-300-500 focus:border-slate-300-500 block px-4 py-1.5  dark:border-gray-300 dark:placeholder-gray-400 dark:text-darkergrey dark:focus:ring-slate-300 dark:focus:border-slate-300 h-24"
+                  placeholder="Type in your comment..."
+                />
+                <div className="mt-5 sm:mt-6 sm:grid sm:grid-flow-row-dense sm:grid-cols-2 sm:gap-3">
+                  <button
+                    type="button"
+                    className="px-12 rounded-md bg-white tracking-[.2em] py-2.5 text-sm font-bcsans text-bcblue shadow-sm ring-1 ring-inset ring-bcblue hover:bg-gray-50 mr-4"
+                    // "mt-3 inline-flex items-center justify-center -md bg-white text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50 sm:col-start-1 sm:mt-0"
+                    onClick={() => setOpen(false)}
+                    ref={cancelButtonRef}
+                  >
+                    CANCEL
+                  </button>
+                  {isLoading ? (
+                    <button
+                      disabled
+                      type="button"
+                      className="inline-flex justify-center rounded-md bg-bcorange/50 px-4 py-2.5 font-bcsans text-bcblue text-sm tracking-[.2em] shadow-sm brightness-110 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 col-start-2"
+                    >
+                      <div
+                        className="mr-2 mt-1 inline-block h-3 w-3 animate-spin rounded-full border-3 border-solid border-current border-r-transparent align-[-0.125em] text-primary motion-reduce:animate-[spin_1.5s_linear_infinite]"
+                        role="status"
+                      >
+                        <span className="!absolute !-m-px !h-px !w-px !overflow-hidden !whitespace-nowrap !border-0 !p-0 ![clip:rect(0,0,0,0)]"></span>
+                      </div>
+                      Loading...
+                    </button>
+                  ) : confirm ? (
+                    <button
+                      type="button"
+                      onClick={handleSubmit}
+                      className="inline-flex justify-center rounded-md bg-bcorange px-4 py-2.5 font-bcsans text-bcblue text-sm tracking-[.2em] shadow-sm hover:brightness-110 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 col-start-2"
```

Comment:
The class names are pretty similar to one in below. Can we `classnames` (or something equivalent) to set different ones according to the condition?
https://www.npmjs.com/package/classnames